slowfast/datasets/decoder.py

```diff
@@ -414,35 +418,49 @@ def pyav_decode(
     else:
```

Comment:
For line 414-417, we should also decode the whole videos and return the frames. Example code could be:
```
        decode_all_video = True
        video_start_pts, video_end_pts = 0, math.inf
        start_end_delta_time = None

        frames = None
        if container.streams.video:
            video_frames, max_pts = pyav_decode_stream(
                container,
                video_start_pts,
                video_end_pts,
                container.streams.video[0],
                {"video": 0},
            )

            frames = [frame.to_rgb().to_ndarray() for frame in video_frames]
            frames = torch.as_tensor(np.stack(frames))
        frames_out = [frames]
```